recipes/mxnet/build.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/env bash
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+   export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
+   if [[ "$target_platform" == "osx-arm64" ]]; then
+       cp config/distribution/darwin_cpu.cmake config.cmake
+   else
+       cp config/distribution/darwin_cpu_mkl.cmake config.cmake
+       export MKL_INCLUDE_DIR=${CONDA_PREFIX}/include
+   fi
+elif [[ "$OSTYPE" == "linux-gnu"* ]]; then
+       # if want to add mkl to cuda builds:
+       # CMAKE_ARGS_BLAS=-DUSE_BLAS='mkl'
+       # export MKL_INCLUDE_DIR=${CONDA_PREFIX}/include
+   if [[ "$cuda_compiler_version" == "None" ]]; then
+       export MKL_INCLUDE_DIR=${CONDA_PREFIX}/include
+       cp config/distribution/linux_cpu_mkl.cmake config.cmake
+   elif [[ $cuda_compiler_version == 11.0 ]]; then
+       cp config/distribution/linux_cu110.cmake config.cmake
+   elif [[ $cuda_compiler_version == 11.2 ]]; then
+       cp config/distribution/linux_cu112.cmake config.cmake  
+   fi
+fi
+
+rm -rf build; mkdir build && cd build
+
+cmake \
+       -DCMAKE_PREFIX_PATH=$PREFIX \
+       -DCMAKE_BUILD_TYPE=Release \
+       -DCMAKE_INSTALL_LIBDIR="lib" \
+       -DCMAKE_INSTALL_PREFIX=$PREFIX \
+       -DCMAKE_INCLUDE_PATH=$PREFIX/include \
+       -DUSE_CPP_PACKAGE=ON \
+       -DUSE_OPENMP=ON \
+       -DBLA_STATIC=OFF \
```

Comment:
```suggestion
       -DBLA_STATIC=OFF \
       -DBUILD_SHARED_LIBS=ON \
```

add -DBUILD_SHARED_LIBS=ON